### ISSUE TITLE
Local Build Fails at Inspector and Watcher does not rebuild on save

### ISSUE THREAD

**Issue by BabylonJSGuide** [BabylonJS/Babylon.js#5944]
Please note that whilst there are failures as described below babylon.max.js is still built and local playground tests are still available as described in the docs. However following any changes to a typescript build the full npm run build has to be done to re-build babylon.max.js.

Using npm version 6.4.1 errors here
```
[12:54:51] Starting 'buildMin'...
i ｢atl｣: Using typescript@3.2.2 from typescript
i ｢atl｣: Using tsconfig.json from C:/Users/User/Documents/GitHub/Babylon.js/inspector/tsconfig.json
i ｢atl｣: Checking started in a separate process...
i ｢atl｣: Time: 13320ms
[12:57:00] 'buildMin' errored after 2.13 min
[12:57:00] Error in plugin "webpack-stream"
Message:
    ./components/sceneExplorer/sceneExplorer.scss (../node_modules/css-loader!../node_modules/sass-loader/lib/loader.js!./components/sceneExplorer/sceneExplorer.scss)
Module build failed (from ../node_modules/sass-loader/lib/loader.js):
Error: Missing binding C:\Users\User\Documents\GitHub\Babylon.js\inspector\node_modules\node-sass\vendor\win32-x64-64\binding.node
Node Sass could not find a binding for your current environment: Windows 64-bit with Node.js 10.x

Found bindings for the following environments:
  - Windows 64-bit with Node.js 8.x

This usually happens because your environment has changed since running `npm install`.
Run `npm rebuild node-sass` to download the binding for your current environment.
    at module.exports (C:\Users\User\Documents\GitHub\Babylon.js\inspector\node_modules\node-sass\lib\binding.js:15:13)
    at Object.<anonymous> (C:\Users\User\Documents\GitHub\Babylon.js\inspector\node_modules\node-sass\lib\index.js:14:35)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.sassLoader (C:\Users\User\Documents\GitHub\Babylon.js\inspector\node_modules\sass-loader\lib\loader.js:46:72)
 @ ./components/sceneExplorer/sceneExplorer.scss 2:14-137
 @ ./components/sceneExplorer/sceneExplorerComponent.tsx
 @ ./inspector.ts
 @ ./index.ts
 @ ./legacy/legacy.ts
./components/embedHost/embedHost.scss (../node_modules/css-loader!../node_modules/sass-loader/lib/loader.js!./components/embedHost/embedHost.scss)
Module build failed (from ../node_modules/sass-loader/lib/loader.js):
Error: Missing binding C:\Users\User\Documents\GitHub\Babylon.js\inspector\node_modules\node-sass\vendor\win32-x64-64\binding.node
Node Sass could not find a binding for your current environment: Windows 64-bit with Node.js 10.x

Found bindings for the following environments:
  - Windows 64-bit with Node.js 8.x

This usually happens because your environment has changed since running `npm install`.
Run `npm rebuild node-sass` to download the binding for your current environment.
    at module.exports (C:\Users\User\Documents\GitHub\Babylon.js\inspector\node_modules\node-sass\lib\binding.js:15:13)
    at Object.<anonymous> (C:\Users\User\Documents\GitHub\Babylon.js\inspector\node_modules\node-sass\lib\index.js:14:35)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.sassLoader (C:\Users\User\Documents\GitHub\Babylon.js\inspector\node_modules\sass-loader\lib\loader.js:46:72)
 @ ./components/embedHost/embedHost.scss 2:14-133
 @ ./components/embedHost/embedHostComponent.tsx
 @ ./inspector.ts
 @ ./index.ts
 @ ./legacy/legacy.ts
./components/actionTabs/actionTabs.scss (../node_modules/css-loader!../node_modules/sass-loader/lib/loader.js!./components/actionTabs/actionTabs.scss)
Module build failed (from ../node_modules/sass-loader/lib/loader.js):
Error: Missing binding C:\Users\User\Documents\GitHub\Babylon.js\inspector\node_modules\node-sass\vendor\win32-x64-64\binding.node
Node Sass could not find a binding for your current environment: Windows 64-bit with Node.js 10.x

Found bindings for the following environments:
  - Windows 64-bit with Node.js 8.x

This usually happens because your environment has changed since running `npm install`.
Run `npm rebuild node-sass` to download the binding for your current environment.
    at module.exports (C:\Users\User\Documents\GitHub\Babylon.js\inspector\node_modules\node-sass\lib\binding.js:15:13)
    at Object.<anonymous> (C:\Users\User\Documents\GitHub\Babylon.js\inspector\node_modules\node-sass\lib\index.js:14:35)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.sassLoader (C:\Users\User\Documents\GitHub\Babylon.js\inspector\node_modules\sass-loader\lib\loader.js:46:72)
 @ ./components/actionTabs/actionTabs.scss 2:14-134
 @ ./components/actionTabs/actionTabsComponent.tsx
 @ ./inspector.ts
 @ ./index.ts
 @ ./legacy/legacy.ts
Details:
    domainEmitter: [object Object]
    domain: [object Object]
    domainThrown: false

[12:57:00] 'inspector' errored after 2.15 min
[12:57:00] 'typescript-libraries' errored after 12 min
[12:57:00] 'typescript-all' errored after 12 min
[12:57:00] 'default' errored after 16 min
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! babylonjs@3.3.0 build: `gulp --max-old-space-size=8192 --tsLintFix`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the babylonjs@3.3.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\User\AppData\Roaming\npm-cache\_logs\2019-02-20T12_57_00_927Z-debug.log

User@User-PC MINGW64 ~/Documents/GitHub/Babylon.js/Tools/Gulp (master)
```

After a file save at 1:04:46 Gulp Run produces the following but no changes in babylon.max.js
```

Built at: 02/20/2019 1:04:41 PM
                     Asset     Size               Chunks                    Chunk Names
    babylonjs.materials.js  394 KiB  babylonjs-materials  [emitted]  [big]  babylonjs-materials
babylonjs.materials.js.map  161 KiB  babylonjs-materials  [emitted]         babylonjs-materials
Entrypoint babylonjs-materials [big] = babylonjs.materials.js babylonjs.materials.js.map
[13:04:41] webpack is watching for changes
i ｢atl｣: Checking started in a separate process...
1:04:46 PM - File change detected. Starting incremental compilation...


1:04:49 PM - Found 0 errors. Watching for file changes.
i ｢atl｣: Checking started in a separate process...
i ｢atl｣: Time: 2ms
[13:05:02] Version: webpack 4.28.4
Built at: 02/20/2019 1:05:02 PM
 2 assets
Entrypoint babylonjs-gui [big] = babylon.gui.js babylon.gui.js.map
[13:05:02] webpack is watching for changes
```

**Comment by BabylonJSGuide** [comment]
After a while the watcher gives these error messages
```
ERROR in ./components/embedHost/embedHost.scss (../node_modules/css-loader!../node_modules/sass-loader/lib/loader.js!./components/embedHost/embedHost.scss)
Module build failed (from ../node_modules/sass-loader/lib/loader.js):
Error: Missing binding C:\Users\User\Documents\GitHub\Babylon.js\inspector\node_modules\node-sass\vendor\win32-x64-64\binding.node
Node Sass could not find a binding for your current environment: Windows 64-bit with Node.js 10.x

Found bindings for the following environments:
  - Windows 64-bit with Node.js 8.x

This usually happens because your environment has changed since running `npm install`.
Run `npm rebuild node-sass` to download the binding for your current environment.
    at module.exports (C:\Users\User\Documents\GitHub\Babylon.js\inspector\node_modules\node-sass\lib\binding.js:15:13)
    at Object.<anonymous> (C:\Users\User\Documents\GitHub\Babylon.js\inspector\node_modules\node-sass\lib\index.js:14:35)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.sassLoader (C:\Users\User\Documents\GitHub\Babylon.js\inspector\node_modules\sass-loader\lib\loader.js:46:72)
 @ ./components/embedHost/embedHost.scss 2:14-133
 @ ./components/embedHost/embedHostComponent.tsx
 @ ./inspector.ts
 @ ./index.ts
 @ ./legacy/legacy.ts

ERROR in ./components/sceneExplorer/sceneExplorer.scss (../node_modules/css-loader!../node_modules/sass-loader/lib/loader.js!./components/sceneExplorer/sceneExplorer.scss)
Module build failed (from ../node_modules/sass-loader/lib/loader.js):
Error: Missing binding C:\Users\User\Documents\GitHub\Babylon.js\inspector\node_modules\node-sass\vendor\win32-x64-64\binding.node
Node Sass could not find a binding for your current environment: Windows 64-bit with Node.js 10.x

Found bindings for the following environments:
  - Windows 64-bit with Node.js 8.x

This usually happens because your environment has changed since running `npm install`.
Run `npm rebuild node-sass` to download the binding for your current environment.
    at module.exports (C:\Users\User\Documents\GitHub\Babylon.js\inspector\node_modules\node-sass\lib\binding.js:15:13)
    at Object.<anonymous> (C:\Users\User\Documents\GitHub\Babylon.js\inspector\node_modules\node-sass\lib\index.js:14:35)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.sassLoader (C:\Users\User\Documents\GitHub\Babylon.js\inspector\node_modules\sass-loader\lib\loader.js:46:72)
 @ ./components/sceneExplorer/sceneExplorer.scss 2:14-137
 @ ./components/sceneExplorer/sceneExplorerComponent.tsx
 @ ./inspector.ts
 @ ./index.ts
 @ ./legacy/legacy.ts

ERROR in ./components/actionTabs/actionTabs.scss (../node_modules/css-loader!../node_modules/sass-loader/lib/loader.js!./components/actionTabs/actionTabs.scss)
Module build failed (from ../node_modules/sass-loader/lib/loader.js):
Error: Missing binding C:\Users\User\Documents\GitHub\Babylon.js\inspector\node_modules\node-sass\vendor\win32-x64-64\binding.node
Node Sass could not find a binding for your current environment: Windows 64-bit with Node.js 10.x

Found bindings for the following environments:
  - Windows 64-bit with Node.js 8.x

This usually happens because your environment has changed since running `npm install`.
Run `npm rebuild node-sass` to download the binding for your current environment.
    at module.exports (C:\Users\User\Documents\GitHub\Babylon.js\inspector\node_modules\node-sass\lib\binding.js:15:13)
    at Object.<anonymous> (C:\Users\User\Documents\GitHub\Babylon.js\inspector\node_modules\node-sass\lib\index.js:14:35)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.sassLoader (C:\Users\User\Documents\GitHub\Babylon.js\inspector\node_modules\sass-loader\lib\loader.js:46:72)
 @ ./components/actionTabs/actionTabs.scss 2:14-134
 @ ./components/actionTabs/actionTabsComponent.tsx
 @ ./inspector.ts
 @ ./index.ts
 @ ./legacy/legacy.ts
[13:08:48] webpack is watching for changes
```

**Comment by deltakosh** [comment]
can you try with node 8?

**Comment by BabylonJSGuide** [comment]
Am using node 10.15.0, now with npm 6.8.0. Think something might have got corrupted during my file changes and new pull requests to local. When I downloaded a new clone of babylon.js to check the polygonMeshBuilder new method I did not get the same errors. Though I did get a load of memory errors when trying Gulp run - probably had too many things open on my laptop. Will close the issue for now as it seems a local problem
